parcels/field.py

```diff
@@ -138,6 +139,17 @@ def __init__(self, name, data, lon=None, lat=None, depth=None, time=None, grid=N
         # since some datasets do not provide the deeper level of data (which is ignored by the interpolation).
         self.data_full_zdim = kwargs.pop('data_full_zdim', None)
 
+    @classmethod
+    def parse_wildcards(cls, filenames):
+        if isinstance(filenames, str):
+            filenames = sorted(glob(str(filenames)))
+        if len(filenames) == 0:
+            raise IOError("Field files not found: %s" % str(filenames))
+        for fp in filenames:
```

Comment:
Is this loop necessary? 
I understand the previous test, but this one isn't already tested later when you try to open the file?

Reply:
This is in case a user provides a list with filenames themselves (rather than wildcards) where not all files actually exist. An unlikely situation probably, but still good to check?

Reply:
Yes I understood, but such problem should be caught in lines 283,284 (new indices), avoiding to loop twice over the files, no?